sdk/generic-sdk-core/generic-core/src/main/java/com/generic/core/http/models/ResponseHandlingMode.java

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.generic.core.http.models;
+
+import java.io.InputStream;
+
+/**
+ * Enum that defines how to handle the body of an HTTP response.
+ */
+public enum ResponseHandlingMode {
```

Comment:
Should this be expandable enum to allow for new values to be added later?

Reply:
What other values do you think we could end up adding in the future?

Reply:
Let's leave as enum for now.